doc/user/content/versions.md

```diff
@@ -33,6 +33,11 @@ main    | [Linux] / [macOS]
 The tarballs for other commits on main can be constructed by replacing
 `latest` in the links above with the full 40-character commit hash.
 
+Unstable Docker images are also available. The most recent published unstable build is published
```

Comment:
Was trying to figure out if there was a way to consolidate this info with the existing table and came up with this:

```diff
diff --git a/doc/user/content/versions.md b/doc/user/content/versions.md
index 6bfaa00e4..e6ffa6a2e 100644
--- a/doc/user/content/versions.md
+++ b/doc/user/content/versions.md
@@ -26,12 +26,14 @@ Binary tarballs are built for every merge to the [main branch on
 GitHub][github]. These tarballs are not suitable for use in production.
 **Run unstable builds at your own risk.**

-Version | Binary tarball links
---------|---------------------
-main    | [Linux] / [macOS]
-
-The tarballs for other commits on main can be constructed by replacing
-`latest` in the links above with the full 40-character commit hash.
+Method  | Available at
+--------|------------
+Tarball | [Linux] / [macOS]
+Docker image | `materialize/materialized:unstable`
+
+To get an arbitrary commit for:
+- **Tarballs**, replace `latest` in the URL with the full 40-character commit hash.
+- **Docker images**, append `-[40-char hash]` to the image name, e.g. `-A1b2C3d4...`

 ## Support
```

Could you work with something like this?

Reply:
That looks great!